.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,87 @@
 return value, etc), please ensure the documentation is also updated to
 reflect this. Documentation is in the `README.md` files of each package. If further documentation is needed please communicate via Github Issues.
 
-## Testing
+### Testing
 
 The packages in Quilt are used in mission-critical production scenarios. As such, we try not to merge any untested code. The coverage doesn't strictly need to be 100% across the board, but testing should remain a primary concern.
 
-## TODO Comments
+### TODO Comments
 
 TODO comments may seem like a great placeholder for work in progress. We prefer top handle this in a different way, using a combination of feature branches and github issues.
 
-### Follow-up Github issues
+#### Follow-up Github issues
 
 If your changes are complete in functionality, but you're not quite happy with auxillary things like documentation or testing, then feel free to make a github issue to track the work that needs to be done. These issues should be linked in the PRs that need a bit more work. This will allow context to be drawn from the code in a more trackable way than a TODO comment. Also, it allows the PR reviewers to see that the documentation or testing is purposefully incomplete and that an appropriate issue exists to track the follow-up work.
 
-### Feature branches
+#### Feature branches
 
 Another option, if you'd like to break work down into reviewable chunks, is to use a feature branch. This would be an initially empty branch that contains the entirety of your feature. Additional units of work can be distributed across several PRs into the feature branch, merged independently, and then the feature branch can be merged as a complete unit into the master branch, when it's ready.
 
-## Releasing a new version
+## Releasing
+
+The release process currently invovles some manual steps to complete. Please ping one of the repo owners in the `#web-foundations` slack channel when you're ready to merge a new PR into `master`, and we will orchestrate a new release.
+
+### For repo owners
+
+**Note** these steps require admin access to the `Shopify/quilt` github repo.
+
+1. Ensure you have the latest `master` branch including all tags:
+
+```
+git checkout master && git pull origin master
+```
+
+2. Begin the release process:
+
+```
+yarn release
+```
+
+3. Follow the prompts to choose a version for each package.
+
+**Note** This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+4. Log in to [Shipit](https://shipit.shopify.io/shopify/quilt/production)
 
-We're working on improving the release experience for new contributors. In the meantime, please ping one of the `quilt` repo owners when you're ready to merge a new PR into `master`, and we will help you with orchestrating a new release via Shipit.
+5. When CI is 🍏 on the tag commit, press `Deploy` to update packages on npm.
 
-This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+## External Contributors
```

Comment:
That's something @BPScott may be able to help with (in regards to the CLA signing process and such).